docs/development/setup.md

```diff
@@ -102,4 +102,27 @@ To remove the APIs from your cluster, simply run
 make uninstall
 ```
 
+**Note** In case `make uninstall` got stuck while deleting `onmetal-system` namespace, first delete all resources
+present in `onmetal-system` namespace and then delete `onmetal-system` namespace using below command.
```

Comment:
**Note** In case `make uninstall` got stuck while deleting `onmetal-system` namespace, delete all resources
 present in `onmetal-system` namespace and then delete `onmetal-system` namespace using the command below:

Reply:
Thanks for reviewing. Addressed.